src/averageAges.js

```diff
@@ -20,6 +20,17 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const menArr = people.filter(person =>
+    (century === undefined)
+      ? person.sex === 'm'
+      : person.sex === 'm' && Math.ceil(person.died / 100) === century
+  );
```

Comment:
```suggestion
  const menArr = people.filter(
  person => century
      ? person.sex === 'm' && Math.ceil(person.died / 100) === century
      : person.sex === 'm'
  );
```
Так лаконичнее, согласись. Да кстати, переносить строку после стрелки - не очень красиво